c29301450.lua

```diff
@@ -107,6 +107,10 @@ end
 function s.retfilter(c,fid)
 	return c:GetFlagEffectLabel(id)==fid
 end
+function s.retchkfilter(c)
+	local tp=c:GetPreviousControler()
+	return Duel.GetLocationCount(tp,LOCATION_MZONE,tp,LOCATION_REASON_TOFIELD)>0
```

Comment:
Could we use `Duel.GetMZoneCount` instead? at  https://github.com/Fluorohydride/ygopro-core/pull/600 we have updated Kaiser check for those functions.

Reply:
changed